tests/api/test_siblings.py

```diff
@@ -368,6 +545,21 @@ def test_node_siblings(client, resource, uid, backward_rel, expected):
         'state': 2,
         'version': '1.7.1'
     })[0]
+    module_build = ModuleKojiBuild.get_or_create({
```

Comment:
How about only creating the module build and its relationships if the `story_type` is `module`. When that is the case, also avoid creating the relationships between the RPM builds and the advisory.